### ISSUE TITLE
Dark background color

### ISSUE THREAD

**Issue by niksy** [chubin/wttr.in#11]
Any chance of explicitly setting background color of output to dark background color? Visuals can be unreadable on light terminal themes.

![screen shot 2016-02-20 at 12 38 17](https://cloud.githubusercontent.com/assets/389286/13196468/d553ffa8-d7ce-11e5-86ea-c9201b5ea771.png)

**Comment by alex179ohm** [comment]
One tip, in order to change colors you could alias the wttr.in command on your .bashrc or .zshrc like the following:
```sh
alias weather='wttr.in/YOUR_CITY | sed -e 's:226m:202m:g'
```
where 226m is the yellow color code on 256 colors terminals which is changed with a more readable orange (202m).

the complete list of colors could be reached [here](http://misc.flogisoft.com/bash/tip_colors_and_formatting#foreground_text1)

**Comment by m0veax** [comment]
You can reduce it to this:

`_weather() {
    tmpfile=$(mktemp)
    curl -s wttr.in/$1 | sed -e 's:226m:202m:g'
}
alias weather='_weather`

**Comment by yckart** [comment]
@m0veax No, not really! What you did is what the first alias from @alex179ohm already does — pipe `curl`'s output to `sed`. But, in addition, each time `_weather` is called you also create a wasteful temporary file (which is not used in your script and even not deleted afterwards).

**Comment by lexicalunit** [comment]
I also use a light terminal theme so I created a shell script to solve this issue.

`weather.sh`:

```
#!/bin/bash

# default location
if [[ -n "$1" ]]; then
    LOC="$1"
else
    LOC="ADD_YOUR_DEFAULT_LOCATION_HERE"
fi

# force dark background color
esc=$'\e'
bg="$esc[48;5;232m"
eol="$esc[K"

echo -e "$bg$eol"
curl -s "http://wttr.in/$LOC" | sed "s/^/$bg$eol  /;s/\[0m/\[0m$bg/g"
echo -e "$bg$eol"
```

<img width="1186" alt="screen shot 2017-01-17 at 4 05 25 pm" src="https://cloud.githubusercontent.com/assets/1903876/22042049/1bd30cac-dccf-11e6-87aa-a8898d9a37ad.png">

**Comment by book** [comment]
The ANSI escape sequences to set a dark background could be simply sent along with the rest of the text, just like the rest of the color escapes.

**Comment by chubin** [comment]
@book The issue not about background color per se, the point is to preare a new color scheme, that will good work with the new background

**Comment by C4K3** [comment]
In case anybody simply wants to remove color entirely, I do that with `sed 's/\x1B[0-9\[;]\+m//g'`

**Comment by chubin** [comment]
@C4K3 Just use ?T
wttr.in/?T

it should work. Does it work for you?

**Comment by C4K3** [comment]
@chubin It does, awesome, I didn't know about that option.

**Comment by ratijas** [comment]
@chubin Hi,

Sorry to ping on this, but could you make min temperature blue color just a little bit brighter, please? It's so cold here, I can barely distinguish the digits at the low end.

![Konsole wttr.in](https://user-images.githubusercontent.com/6737986/102731578-37a27780-4349-11eb-8949-ae928c93e788.png)

I'm using Konsole with slightly customized Arc Dark theme, its background color is solid `#383c4a`.

**Comment by chubin** [comment]
I've changed it to violet. I think it is better so. You know, if it is -56⁰C outside, I think blue is not blue enough for that, it should be something special

![Selection_083](https://user-images.githubusercontent.com/3875145/103443688-05a2e680-4c62-11eb-84ab-9f77e3761d95.png)

**Comment by Layerex** [comment]
I made this utility a while ago primarily to deal with that problem. https://github.com/Layerex/term-invert-colors
![screen](https://user-images.githubusercontent.com/59440032/119017100-5150f500-b9a3-11eb-9736-b1911f3369e8.png)

**Comment by chubin** [comment]
@Layerex It is a cool tool. Maybe it we should integrate it on the server side

**Comment by Layerex** [comment]
@chubin would be cool. But, I guess, fine-tuning original colorscheme without full inversion would be much better. For example, making sun orange or red and darkening other colors would fix the problem of readability without significantly altering colors.

**Comment by chubin** [comment]
@Layerex I agree. I believe we should just create color schemes, and make it possible for the uses to add their own color schemes + select the current color scheme

**Comment by hello-smile6** [comment]
Why not use a proxy to invert the colors?

**Comment by chubin** [comment]
What proxy do you mean?

It would be really a good feature, to be able to specify the color scheme on the server side,
so that anybody could just use it

**Comment by hello-smile6** [comment]
> What proxy do you mean?

One that doesn't exist yet.

**Comment by hello-smile6** [comment]
@chubin https://ansi-invert-proxy.9pfs.repl.co/https:/wttr.in/
I made a proxy that uses @Layerex 's library to invert ANSI escape sequences for requests. It's currently slow, but I'll try to make it more reliable (and maybe add caching).
